components/concept/engine/src/main/java/mozilla/components/concept/engine/ClearDatable.kt

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.concept.engine
+
+/**
+ * Contract to indicate how objects able to clear browser data should behave.
+ */
+@Suppress("TooManyFunctions")
+interface ClearDatable {
```

Comment:
I think the phrase that you're looking for may be `DataClearable`? Data is not an actionable verb so an adjective form looks rather odd 🤔

Reply:
Agree, thanks!

Reply:
Android studio was complaining about `DataClearable` is suggested me `DataCleanable`, I think is OK too.